src/parser/esprima_cpp/esprima.cpp

```diff
@@ -131,6 +131,7 @@ struct Config : public gc {
     bool tokens : 1;
     bool comment : 1;
     bool parseSingleFunction : 1;
+    bool repaseArguments : 1;
```

Comment:
It's a typo. Please fix `repaseArguments` to `reparseArguments`.

Reply:
Oopp, thanks! :D